modules/apps/commerce/commerce-frontend-js/test/components/Diagram/Diagram.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {cleanup, render} from '@testing-library/react';
+import React from 'react';
+
+import Diagram from '../../../src/main/resources/META-INF/resources/components/Diagram/Diagram';
```

Comment:
@garofalog 
This is most probably having unit tests fail for the `'[..]/components/Diagram/Diagram'` dir name case... which is wrong (it should be `'../../../src/main/resources/META-INF/resources/components/diagram/Diagram'`). On MacOS you'd probably wouldn't notice it, as the FS is not case sensitive. But other *nix OS's (including our CI) are.

Can you please check this out?

Reply:
Yay, they passed. :)

![image](https://user-images.githubusercontent.com/49437061/117977687-61c4f800-b331-11eb-8777-30002be3c3bf.png)